test/EFCore.Sqlite.FunctionalTests/Query/NorthwindFunctionsQuerySqliteTest.cs

```diff
@@ -856,6 +856,49 @@ public override async Task String_Contains_Column(bool async)
 """);
     }
 
+    public override async Task String_Contains_in_projection(bool async)
+    {
+        await base.String_Contains_in_projection(async);
+
+        AssertSql(
+            """
+SELECT "c"."CustomerID" AS "Id", CASE
+    WHEN instr("c"."CompanyName", "c"."ContactName") > 0 THEN 1
+    ELSE 0
+END AS "Value"
+FROM "Customers" AS "c"
+""");
+    }
+
+    public override async Task String_Contains_negated_in_predicate(bool async)
+    {
+        await base.String_Contains_negated_in_predicate(async);
+
+        AssertSql(
+            """
+SELECT "c"."CustomerID", "c"."Address", "c"."City", "c"."CompanyName", "c"."ContactName", "c"."ContactTitle", "c"."Country", "c"."Fax", "c"."Phone", "c"."PostalCode", "c"."Region"
+FROM "Customers" AS "c"
+WHERE NOT (CASE
+    WHEN instr("c"."CompanyName", "c"."ContactName") > 0 THEN 1
+    ELSE 0
+END)
```

Comment:
ideally this should be:

Reply:
```suggestion
WHERE CASE
    WHEN instr("c"."CompanyName", "c"."ContactName") > 0 THEN 0
    ELSE 1
END
```
i.e. we could remove the `NOT` by distributing it over the `CASE`.
Can we do better?

Reply:
Good idea, filed https://github.com/dotnet/efcore/issues/33857. We also have https://github.com/dotnet/efcore/issues/25977 around case nullability, but it's much more complicated and arguably not that useful.